src/server/routes/api.js

```diff
@@ -138,6 +140,17 @@ module.exports = function(app) {
         });
     });
 
+  api.get('/admin/generateTeams/:eventAlias', requireAuth, roleAuth(roles.ROLE_ADMIN),
+    isOrganiser, (req, res) => {
+      return User.find({event: req.event, deleted: false, sanitized: true})
+        .populate('account')
+        .exec()
+        .catch(err => Errors.respondError(res, err, Errors.DATABASE_ERROR))
+        .then((users) => {
+          return res.send(buildVertices(users));
```

Comment:
You'll have to do a little more here - look at https://github.com/UCSDTESC/Check-in/blob/master/src/server/routes/api.js#L132